type-reducer/standard_rename_only_mapped_names.txt

```diff
@@ -0,0 +1,14 @@
+### Rename only
+GRPCRouteRules->GRPCRouteRule
+HTTPRouteRules->HTTPRouteRule
+HTTPRouteRulesFilters->HTTPRouteFilter
+GRPCRouteRulesMatches->GRPCRouteMatch
+HTTPRouteRulesMatches->RouteMatch
+HTTPRouteRulesTimeouts->HTTPRouteTimeout
+GRPCRouteRulesBackendRefs->GRPCBackendReference
+HTTPRouteRulesBackendRefs->HTTPBackendReference
+GRPCRouteRulesMatchesMethod->GRPCMethodMatch
+HTTPRouteRulesMatchesMethod->HTTPMethodMatch
+HTTPRouteRulesBackendRefsFilters->HTTPRouteBackendFilter
+HTTPRouteRequestRedirect->RequestRedirect
+HTTPRouteRulesMatchesPath->PathMatch
```

Comment:
Again, wondering if we can avoid checking in the mapping files? WDYT? :thinking:

Reply:
We need this information unfortunately, it shows how to map Kopium dervied type names to new, friendlier names.